admin_manual/installation/nginx_configuration.rst

```diff
@@ -80,8 +80,10 @@ Nginx Configuration
       rewrite ^/.well-known/host-meta /public.php?service=host-meta last;
       rewrite ^/.well-known/host-meta.json /public.php?service=host-meta-json last;
```

Comment:
Ok, nevermind. This is only needed for the user_webfinger app (https://github.com/owncloud/apps/tree/master/user_webfinger)